config-linux.md

```diff
@@ -494,6 +494,21 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.  The OCI runtime must ensure that the needed cgroup controllers are enabled for the cgroup.
```

Comment:
> The OCI runtime must ensure that the needed cgroup controllers are enabled for the cgroup.

s/must/SHOULD/ ?
What is the expected behavior if the runtime cannot enable the controller?

Reply:
I guess it should error out?

Reply:
It should error out because it cannot fulfill the configuration that was requested.

Reply:
"MUST error out" sounds too strict for rootless containers

Reply:
rootless containers should not get a cgroup configuration if they are not able to satisfy it.  In what cases would the cgroup configuration be set if it is not usable?

Reply:
If rootless containers under `runc` today aren't giving errors if you pass them a cgroup configuration they cannot fulfil, then that's a bug in `runc`. It's a hard requirement of the runtime-spec that runtimes must not silently ignore any aspect of the configuration.